nautobot/extras/jobs.py

```diff
@@ -1211,7 +1214,22 @@ def _run_job():
             with transaction.atomic():
                 # Script-like behavior
                 job.active_test = "run"
-                output = job.run(data=data, commit=commit)
+                if profile:
+                    import cProfile
+
+                    # TODO: This should probably be available as a file download rather than dumped to the hard drive.
+                    # Pending this: https://github.com/nautobot/nautobot/issues/3352
+                    profiling_path = f"/tmp/{job_result_pk}.pstats"
```

Comment:
Perhaps?

```suggestion
                    profiling_path = f"/tmp/nautobot-jobresult-{job_result_pk}.pstats"
```

If so be sure to update the docs as well :)